modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -202,7 +197,7 @@ $ oc apply -f ingress-autoscaler.yaml
 +
 [source,terminal]
 ----
-$ oc get ingresscontroller/default -o yaml | grep replicas:
+$ oc get -n openshift-ingress-operator ingresscontroller/default -o yaml | grep replicas:
 ----
 +
 .Example output
```

Comment:
In line 206, better to add two spaces before replicas, so it can be:
`  replicas: 3`

Reply:
Updated.